programmers/난이도별/level01.2016년/easyearn77.cpp

```diff
@@ -0,0 +1,15 @@
+#include <string>
+#include <vector>
+
+using namespace std;
+
+string solution(int a, int b) {
+    const string dow[7]={"THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"};
+    const int dom[12]={31,29,31,30,31,30,31,31,30,31,30,31};
```

Comment:
```suggestion
    const string dow[]={"THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"};
    const int dom[]={31,29,31,30,31,30,31,31,30,31,30,31};
```
배열 초기화할 때 값을 정의해주지 않으면 자동으로 할당됩니다!